### PR TITLE
feat: remove featured articles

### LINKED ISSUE #10426: Featured articles - remove
# HI 
This section is outdated, taking space on landing page

Revisit in q3 with some educative content, or some gen art thingies

anyway, if we decide to go with something, it probably wont on the end of landing page, 3 cards

## some stats: 
image

one click in 30 days, nice 

say bye

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10426

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [] My fix has changed UI

### CODE REVIEW

**Review by hassnian — APPROVED** [review]

File: components/landing/FeaturedArticles.vue [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-      target="_blank"
-      rel="nofollow noopener noreferrer">
-      {{ $t('helper.seeMore') }} >
-    </a>
-  </div>
-</template>
-
-<script lang="ts" setup>
-import { CardArticle } from '@kodadot1/brick'
-import posts from '@/script/posts.json'
```
Comment (Jarsen136) [inline components/landing/FeaturedArticles.vue:29]: Let's also remove these files `script/posts.mjs` and `script/posts.json` if they are not used

Comment (hassnian) [reply components/landing/FeaturedArticles.vue:29]: what about `CardArticle`, should also remove that ?

Comment (prachi00) [reply components/landing/FeaturedArticles.vue:29]: Screenshot 2024-06-09 at 12 41 37 AM
@Jarsen136 i see .mjs being used here in this script, and script/posts.json is used inside mjs, do we need this script? I am not sure

Comment (Jarsen136) [reply components/landing/FeaturedArticles.vue:29]: > @Jarsen136 i see .mjs being used here in this script, and script/posts.json is used inside mjs, do we need this script? I am not sure

No, please remove all of them.

Comment (prachi00) [reply components/landing/FeaturedArticles.vue:29]: done

**Review by hassnian — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
     "locales:check": "node ./script/checkLocale.mjs",
     "start": "nuxi preview",
     "start:node": "PORT=9090 node .output/server/index.mjs",
-    "generate": "node ./script/posts.mjs && cross-env NODE_OPTIONS=--openssl-legacy-provider nuxi build",
```
Comment (Jarsen136) [inline package.json:29]: Only need to delete the `posts` related.
```
"generate": "cross-env NODE_OPTIONS=--openssl-legacy-provider nuxi build",
```

**Review by Jarsen136 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -26,15 +26,15 @@
...
     "lint": "eslint --cache --ignore-path .gitignore --ext .js,.ts,.vue .",
     "lint:quiet": "eslint --cache --quiet --ignore-path .gitignore --ext .js,.ts,.vue .",
     "lint:fix": "eslint --cache --fix --quiet --ignore-path .gitignore --ext .js,.ts,.vue .",
     "test": "vitest run --reporter verbose --allowOnly",
     "test:watch": "vitest --reporter verbose",
     "test:coverage": "vitest run --coverage",
     "test:e2e": "playwright test --ui",
-    "prepare": "pnpm -F static build && nuxi prepare && husky install"
+    "prepare": "pnpm -F static build && nuxi prepare && husky install",
+    "generate": "cross-env NODE_OPTIONS=--openssl-legacy-provider nuxi build"
```
Comment (Jarsen136) [inline package.json:37]: Could you move it to the same position as before? (move upper)

**Review by Jarsen136 — APPROVED** [review]
Otherwise lgtm

**Pull request merged**